packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.scss

```diff
@@ -80,13 +80,12 @@
 
     &__items {
         position: absolute;
-        top: 100%;
+        top: 4.8rem;
         width: 100%;
         display: flex;
         flex-direction: column;
         align-items: flex-start;
         z-index: 2;
-        margin-top: -1.2rem;
         border-radius: 0.4rem;
```

Comment:
Position the dropdown options list in relation to the top exclusively, given that it's always shifted to the bottom by the dropdown height + 8px, counting from the dropdown's top